android/source/VideoLocker/src/org/edx/mobile/view/VideoListActivity.java

```diff
@@ -125,7 +125,15 @@ protected void onSaveInstanceState(Bundle outState) {
     @Override
     protected void onRestoreInstanceState(Bundle savedInstanceState) {
         if (savedInstanceState != null) {
-            super.onRestoreInstanceState(savedInstanceState);
+            //FIXME: There is an error of null pointer exception on some devices when super function is called.
+            //This is a workaround where the crash is handled in
+            //try catch and hence the app does not crash
+            try{
+                super.onRestoreInstanceState(savedInstanceState);
+            }catch(Exception e){
+                e.printStackTrace();
+                LogUtil.error("player_issue","Null pointer exception on onRestoreInstanceState");
```

Comment:
Yes we are going to create a new PR to change all the stack trace messages to use built in log functions. We wanted to add all the stack trace changes in one PR and hence we have used stack trace for this change and will update it in the PR for changing logs.
Will that be ok?